Bugfix unicode compatibility

wradstok opened the pull request:
Hi,

When operating on my data set I was getting the following error:

```
> Traceback (most recent call last):
>   File "codes/run.py", line 361, in <module>
>     main(parse_args())
>   File "codes/run.py", line 211, in main
>     train_triples = read_triple(os.path.join(args.data_path, 'train.txt'), entity2id, relation2id)
>   File "codes/run.py", line 127, in read_triple
>     triples.append((entity2id[h], relation2id[r], entity2id[t]))
> KeyError: 'Găgăuzia\xa0'
```

To fix the issue, I added calls to `unicodedata.normalize()` before loading triples to normalize such weird characters (non-breaking spaces). I also fixed two minor grammatical mistakes in error messages.

cthoyt commented on codes/run.py:
```diff
@@ -123,7 +124,7 @@ def read_triple(file_path, entity2id, relation2id):
     with open(file_path) as fin:
         for line in fin:
-            h, r, t = line.strip().split('\t')
+            h, r, t = map(lambda x: x.strip(), unicodedata.normalize('NFKC', line).split('\t'))
```
`str.strip` could be a shorter replacement for `lambda x: x.strip()`

wradstok replied:
Thanks, didn't realize this was possible.

wradstok commented:
I mistakenly thought I was dealing with a unicode issue due to the error I received. Upon investigating closer I realized that when the entity/relations are loaded and the entire line is split, trailing spaces are removed because the names are at the end. However, when loading triples this only occurs on the tail entities. Fixed by mapping str.split() on all components when loading triples.
